enterprise/config/buildbuddy.workflow.yaml

```
@@ -0,0 +1,44 @@
+# Run with:
+# bazel run //enterprise/server -- --config_file=enterprise/config/buildbuddy.workflow.yaml --config_secrets.provider=gcp --config_secrets.gcp.project_id=flame-build
+app:
+  no_default_user_group: true
+  create_group_per_user: true
+  add_user_to_domain_group: true
+  code_editor_enabled: true
+  default_redis_target: "localhost:6379"
+  enable_target_tracking: true
+  user_owned_keys_enabled: true
+  # The workflow runner cannot communicate with the local app server via localhost
+  # because they have different network stacks. Instead you must find and hardcode
+  # the IP address of the machine the app server is running on
+  cache_api_url: "grpc://TODO:1985"
+  events_api_url: "grpc:/TODO:1985"
+  remote_execution_api_url: "grpc:/TODO:1985"
```

doesn't matter a whole lot either way since these are TODO, but missing a `/` here

```suggestion
  events_api_url: "grpc://TODO:1985"
  remote_execution_api_url: "grpc://TODO:1985"
```